### DISCUSSION TITLE
Allow SyncDict and SyncList to have their size initialized like a standard list and dictionary

### DISCUSSION THREAD

**Discussion by sinfulspartan** [FirstGearGames/FishNet#291] [Feature Request]
![image](https://user-images.githubusercontent.com/89873705/224493874-4ef50302-f329-4f2e-856b-1a14bd98e45f.png)

**Comment by FirstGearGames** [comment]
While possible, this would require a bit of a refactor on the synclist,hashset, ect. I do like the idea but at this time I'm going to have to decline it given more pressing things needing attention.

And yes, I realize I asked you to make it a suggestion. I did plan to initially include it but after thinking it over I'd prefer to put the time elsewhere now.
